sdk/appconfiguration/app-configuration/samples-dev/featureFlag.ts

```diff
@@ -13,6 +13,7 @@ import {
   FeatureFlagValue,
   parseFeatureFlag
 } from "@azure/app-configuration";
+import { isArray } from "util";
```

Comment:
Array.isArray is built in to JavaScript. I'd use that over pulling in something from a module.

Reply:
Done!

Reply:
Great catch!